### DISCUSSION TITLE
Malicious Repositories

### DISCUSSION THREAD

**Discussion by cydave** [community/community#133657] [Repositories] [answered]
Hello \o

Looks like Github has a bit of a malware / spam account problem.
Could any hubber of the git check those out?

https://gist.github.com/cydave/9c185d3c32c42ffb1e3bac72806f5afb

**Comment by queenofcorgis** [comment, marked as answer]
Hi @cydave 👋🏼  We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. I am going to close this post, but for this and any future incidents, please refer to those links.

Thank you!
